spec/controllers/redirections_controller_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+describe RedirectionsController do
```

Comment:
A nit, but the preferred new syntax is `Rspec.describe BlahClassName do` ([link to docs](https://relishapp.com/rspec/rspec-core/v/3-0/docs/configuration/global-namespace-dsl)).